content/docs/implementation-notes.md

```diff
@@ -9,175 +9,174 @@ redirect_from:
-
-You can imagine this process as a pseudocode:
+Anda dapat membayangkan proses ini sebagai *pseudocode*:
 
 ```js
 function isClass(type) {
-  // React.Component subclasses have this flag
+  // React.Component *subclasses* memiliki *flag* ini
   return (
     Boolean(type.prototype) &&
     Boolean(type.prototype.isReactComponent)
   );
 }
 
-// This function takes a React element (e.g. <App />)
-// and returns a DOM or Native node representing the mounted tree.
+// Fungsi ini mengambil elemen React (misalnya <App />)
+// dan mengembalikan DOM atau *node* Native yang mewakili diagram yang dipasang.
 function mount(element) {
   var type = element.type;
   var props = element.props;
 
-  // We will determine the rendered element
-  // by either running the type as function
-  // or creating an instance and calling render().
+  // Kita akan menentukan elemen yang di *render*
+  // dengan menjalankan tipe sebagai fungsi
+  // atau membuat *instance* dan memanggil render().
   var renderedElement;
   if (isClass(type)) {
-    // Component class
+    // Komponen kelas
     var publicInstance = new type(props);
-    // Set the props
+    // Atur *props*
     publicInstance.props = props;
-    // Call the lifecycle if necessary
+    // Panggil *lifecycle* jika diperlukan
     if (publicInstance.componentWillMount) {
       publicInstance.componentWillMount();
     }
-    // Get the rendered element by calling render()
+    // Dapatkan elemen yang di *render* dengan memanggil render()
     renderedElement = publicInstance.render();
   } else {
-    // Component function
+    // Komponen fungsi
     renderedElement = type(props);
   }
 
-  // This process is recursive because a component may
-  // return an element with a type of another component.
+  // Proses ini rekursif karena komponen mungkin
+  // mengembalikan elemen dengan tipe komponen lain.
   return mount(renderedElement);
 
-  // Note: this implementation is incomplete and recurses infinitely!
-  // It only handles elements like <App /> or <Button />.
-  // It doesn't handle elements like <div /> or <p /> yet.
+  // Catatan: implementasi ini tidak lengkap dan berulang-ulang!
+  // Ini hanya menangani elemen seperti <App /> atau <Button />.
+  // Ini tidak menangani elemen seperti <div /> atau <p /> belum.
 }
 
 var rootEl = document.getElementById('root');
 var node = mount(<App />);
 rootEl.appendChild(node);
 ```
 
->**Note:**
+>**Catatan:**
 >
->This really *is* a pseudo-code. It isn't similar to the real implementation. It will also cause a stack overflow because we haven't discussed when to stop the recursion.
+>Ini benar-benar *pseudo-code*. Itu tidak mirip dengan implementasi nyata. Itu juga akan menyebabkan *stack overflow* karena kita belum membahas kapan harus menghentikan rekursi.
```

Comment:
```suggestion
>Implementasi di atas benar-benar *pseudo-code*, dan tidak mirip dengan implementasi nyata. Implementasi ini juga akan menyebabkan *stack overflow* karena kita belum membahas kapan harus menghentikan rekursi.
```

Do make note of ambiguous uses of "itu". What does "itu" refer to?

In this case it applies to the above implementation, so I fixed this note to reflect that.

Reply:
Whoa,.  I see ,OK thanks